source/_drafts/A-Virtual-Year-Studying-Data-Science-online.md

```diff
@@ -0,0 +1,20 @@
+---
+title: 'A Virtual Year, Studying Data Science Online'
+categories:
+  - [general]
+  - [education]
+author: Di Yao
+number: 11
+tags:
+---
+2020 has been a year filled with unprecedented challenges, especially for students and universities. My studies in [Master of Data Science](https://www.uwa.edu.au/study/courses/master-of-data-science) at the University of Western Australia have been moved online and we had to adapt to these changes quickly. 
+
+Flexibility is the best part about studying online, as most lectures and labs are recorded so it gives us the flexibility to watch them at any time I want. There are also multiple lab sessions for each subject, and I could attend as many as I needed to ask questions. I personally have found that the lab sessions have become more helpful and more collaborative, I could learn from other students asking their questions and come across concepts that I have not considered before.
+
+Although a more flexible schedule with classes has enabled me to better organise studies and other commitments, it also became an issue when I started to think that lectures and labs can be caught up later. When other commitments took priority and by the time I realised how many classes I am behind, there were just too many to catch up. Therefore, it is important to set priorities straight when studying online and ensure you can finish the goals you set for yourself each week. 
+
+The weirdest thing would be doing exams at home, especially with open book exams where they are not monitored. These exams are done within the scheduled period and submitted by us online.  Whereas closed book exams are being monitored through Examplify, which has the same exam conditions as we write exams on campus. 
+
+Both the university and staff member had put in enormous effort to support students through the transition to online studies. Many professors had organised additional consultations hours to assist students and giving as much guidance as they could to ensure that we have a good understanding about concepts and assignment tasks. We may find there are areas to be improved in this process, but we need to understand that the professors are doing this first time as well. 
+
+I had enjoyed the past virtual year with my studies and learnt so much more about Data Science every day. So hopefully another virtual year in 2021 can teach us all something new! 
```

Comment:
I enjoyed the past virtual year with my studies and learnt so much more about Data Science every day. So hopefully 2021 can teach us all something new!